WebApplication1/Model/SumAccumulation.cs

```diff
@@ -7,14 +7,16 @@ public static class SumAccumulation
         #region fields
 
         private static int sum;
-        private static Dictionary<string, CountExists> casheDictionary = new Dictionary<string, CountExists>();
+        private static Dictionary<string, CountExists> cacheDictionary = new Dictionary<string, CountExists>();
+        private static string keyword = "the";
```

Comment:
Keyword should be set from the UI or from config file in the future and should not be hard coded. The correspondent issue will be added.